packages/provider/waallet/content/provider.ts

```diff
@@ -18,28 +22,29 @@ export class WaalletContentProvider extends EventEmitter {
     return res as T
   }
 
-  // Demo code
-  // Usage:
+  // Usage in dapp site:
   //   await (window as any).waallet.createWindow({
-  //     user: "imToken Labs",
-  //     challengeBase64Url: "Pezpmh0P8lXu3tP-V5wSIGnHNVvns0EWyloweQTjmE8",
-  //     authAttach: "cross-platform",
+  //     creation: {
+  //       user: "imToken Labs",
+  //       challenge: "5r264oeeza45DAAnFgSNLybypGsY64GeIa2C5UqbmRk",
+  //     },
+  //     request: {
+  //       credentialId: "jyZ19cHuw8toyyZDHxz7dOVmZ00fRSsvm1WSMV9dfRc",
+  //       challenge: "5r264oeeza45DAAnFgSNLybypGsY64GeIa2C5UqbmRk",
+  //     },
   //   });
   private async createWindow({
-    user,
-    challengeBase64Url,
-    authAttach
+    creation: creation,
```

Comment:
Q: What is the `creation` type?

Reply:
The `creation` variable, representing WebAuthn arguments during creation, is mapped to the `WebauthnCreation` type.
Demo code for requesting WebAuthn from the Dapp website is provided.
However, there is a bug doing not allow `creation` to be undefined. The next commit will fix this bug.